aquacomputer_d5next.c

```diff
@@ -621,11 +664,11 @@ struct aqc_data {
 	u32 power_cycles;
 
 	/* Sensor values */
-	s32 temp_input[20];	/* Max 4 physical and 16 virtual or 8 physical and 12 virtual */
-	s32 speed_input[8];
-	u32 speed_input_min[8];
-	u32 speed_input_target[8];
-	u32 speed_input_max[8];
+	s32 temp_input[40];	/* Max 4 physical + 16 virtual + 20 aquabus or 8 physical + 12 virtual + 20 aquabus */
+	s32 speed_input[20];	/* Max 8 physical + 12 aquabus */
+	u32 speed_input_min[20];
+	u32 speed_input_target[20];
```

Comment:
This array (`speed_input_target`) is only used for one channel of the Leakshield. It's value should always have been `1`, not `8`. I'll fix it separately, but please remove this change from this PR.